libasn1fix/asn1fix_cws.c

```diff
@@ -307,6 +307,8 @@ _asn1f_assign_cell_value(arg_t *arg, struct asn1p_ioc_row_s *row, struct asn1p_i
 			FATAL("Cannot find %s referenced by %s at line %d",
 				p, arg->expr->Identifier,
 				arg->expr->_lineno);
+			asn1p_ref_free(ref);
```

Comment:
OK, I think I straightened it out.

1. Do you think it's a good idea to have `free(p);` before `WARNING("blah-blah %s", p);`? I've seen this in at least one more place (fixed it, of course). 
2. Understood. But in the future please make it clear and explicit - PRs that can stand independently from each other is a good idea, but PRs that conflict is not.